src/test/java/api/items/ItemApiExamples.java

```diff
@@ -381,6 +404,12 @@ public void canUpdateExistingItem()
     assertThat(getResponse.getStatusCode(), is(200));
     JsonObject updatedItem = getResponse.getJson();
 
+    assertThat(updatedItem.containsKey(Item.TAGS_KEY), is(true));
+    final JsonObject tags = updatedItem.getJsonObject(Item.TAGS_KEY);
```

Comment:
Personally, I might include whitespace between these three blocks of code, to better separate them